{{cookiecutter.extension_name}}/setup.py

```diff
@@ -26,6 +26,7 @@ class NodeModulesMissing(Exception):
     packages             = ['{{ cookiecutter.extension_name }}'],
     author               = '{{ cookiecutter.author_name }}',
     author_email         = '{{ cookiecutter.author_email }}',
+    keywords             = 'jupyterlab extension',
```

Comment:
I would prefer having at least `jupyterlab` also by itself - maybe 'jupyterlab' and 'jupyterlab extension'?